README.md

```diff
@@ -69,7 +69,7 @@ Install all non-ROS dependencies of OpenRMF packages,
 sudo apt update && sudo apt install \
   git cmake python3-vcstool curl \
   -y
-python3 -m pip install flask-socketio fastapi uvicorn datamodel_code_generator
+python3 -m pip install flask-socketio fastapi websockets uvicorn datamodel_code_generator
```

Comment:
If a user had previously installed `websockts` as an Ubuntu system package via `apt install python3-websockets`, this `pip` install command will not install the latest version of websockts v10.4.
```
yadu@yadu:~$ dpkg -l | grep python3-websockets
ii  python3-websockets                                 9.1-1                                   all          implementation of the WebSocket Protocol (RFC 6455)
```
```
yadu@yadu:~$ python3 -m pip install websockets
Defaulting to user installation because normal site-packages is not writeable
Requirement already satisfied: websockets in /usr/lib/python3/dist-packages (9.1)
```

So I think the instructions should be updated such that the apt version is purged first if preset and then installed via pip. ie add this line before install all the packages via pip. 
```
sudo apt purge python3-websockets
```

but this would also require us to remove [this line](https://github.com/open-rmf/rmf_demos/blob/459919bb8617a8344a7d456c3c054baca6b39b68/rmf_demos_panel/package.xml#L13) from the `package.xml` of `rmf_demos_panel`. Else, when running the rosdep command, it will also install `python3-websockets`.

Reply:
@Yadunud pinging again